nifi-marklogic-processors/src/main/java/org/apache/nifi/marklogic/processor/QueryRowsMarkLogic.java

```diff
@@ -19,35 +20,40 @@
 import java.io.InputStream;
 import java.util.*;
 
-@Tags({"MarkLogic", "Query", "Read", "Rows"})
+@Tags({ "MarkLogic", "Query", "Read", "Rows" })
```

Comment:
Were you able to use the .editorconfig file in your editor? I want to avoid all these whitespace changes as it makes analyzing changesets much more difficult.